specs/core/0_beacon-chain.md

```diff
@@ -1570,19 +1570,16 @@ def lmd_ghost(store: Store, start_state: BeaconState, start_block: BeaconBlock)
     Execute the LMD-GHOST algorithm to find the head ``BeaconBlock``.
     """
     validators = start_state.validator_registry
-    active_validators = [
-        validators[i]
-        for i in get_active_validator_indices(validators, start_state.slot)
-    ]
+    active_validator_indices = get_active_validator_indices(validators, start_state.slot)
     attestation_targets = [
-        get_latest_attestation_target(store, validator)
-        for validator in active_validators
+        (validator_index, get_latest_attestation_target(store, validator_index))
+        for validator_index in active_validator_indices
     ]
 
     def get_vote_count(block: BeaconBlock) -> int:
-        return len([
-            target
-            for target in attestation_targets
+        return sum([
+            int(start_state.validator_balances[validator_index])
```

Comment:
This should be `get_effective_balance(start_state.validator_balances[validator_index])`

```suggestion
            int(get_effective_balance(start_state.validator_balances[validator_index]))
```